src/vendor/TypingDNA.js

```diff
@@ -0,0 +1,1258 @@
+/**
+ * TypingDNA - Typing Biometrics JavaScript API
+ * https://api.typingdna.com/scripts/typingdna.js
+ * https://typingdna.com/scripts/typingdna.js (alternative)
+ *
+ * @version 2.15
+ * @author Raul Popa
+ * @copyright SC TypingDNA SRL, http://typingdna.com
+ * @license http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@rchild-okta  I had talked about this with you earlier. Wanted to double check if it is okay to add this library here. Should we notify legal. Mau was mentioning that we maintain a list of things external libs we use under a particular license and we might want to add this to that list.

Reply:
Checked with Reman looks like apache is whitelisted and we dont have to worry about this. Also Raghu had talked with Security to check if it is okay to use this library. Will check with both again before we go beta.